apps/ideaspace/src/components/common/DialogBox/DialogBoxButton.js

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import { atoms } from '@devlaunchers/components/src/components';
+
+const DialogBoxButton = ({ handleConfirmButton, handleCancelButton, buttonDetail }) => {
+  // when two buttons are same type, first onClick is Confirm, second is Cancel
+  // when only one button, onClick is Cancel.
+  if (buttonDetail[0] == 'alternative primary') {
+    return (
+      <>
+        <atoms.Button
+          buttonSize='standard'
+          buttonType='alternative'
+          onClick={handleCancelButton}
+        >
+          {buttonDetail[1]}
+        </atoms.Button>
+        <atoms.Button
+          buttonSize='standard'
+          buttonType='primary'
+          onClick={handleConfirmButton}
+        >
+          {buttonDetail[2]}
+        </atoms.Button>
+      </>
+    )
+  } else if (buttonDetail[0] == 'primary alternative') {
+    return (
+      <>
+        <atoms.Button
+          buttonSize='standard'
+          buttonType='primary'
+          onClick={handleConfirmButton}
+        >
+          {buttonDetail[1]}
+        </atoms.Button>
+        <atoms.Button
+          buttonSize='standard'
+          buttonType='alternative'
+          onClick={handleCancelButton}
+        >
+          {buttonDetail[2]}
+        </atoms.Button>
+      </>
+    )
+  } else if (buttonDetail[0] == 'primary primary') {
+    return (
```

Comment:
Could we extract this to a component or function? It looks like this is shared code with the conditional case above